test/rules/import-blacklist-submodules/test.ts.lint

```diff
@@ -0,0 +1,25 @@
+import { submodule } from '@blueprintjs/core';
+
+import { submodule } from '@blueprintjs/core/sub-module';
+                                            ~~~~~~~~~~~ [0]
+
+import submodule = require('@blueprintjs/core');
+
+import submodule = require('@blueprintjs/core/sub-module');
+                                             ~~~~~~~~~~~ [0]
+
+import { submodule } from '@blueprintjs/core';
+
+import { submodule } from '@blueprintjs/core/a/b/c/sub-module';
+                                            ~~~~~~~~~~~~~~~~~ [0]
+
+import sub-module = require('@blueprintjs/core');
+
+import sub-module = require('@blueprintjs/core/a/b/c/sub-module');
+                                              ~~~~~~~~~~~~~~~~~ [0]
+
+// non-static imports cannot be checked
+import {Observable} from rxjs;
```

Comment:
I was confused by this as well. I relied heavily on the [import-blacklist test code](https://github.com/palantir/tslint/blob/master/test/rules/import-blacklist/test.ts.lint) to get my test case up and running. Could use some clarification on this point.